SafeguardPasswords/Scripts/Bootstrap.ps1

```diff
@@ -0,0 +1,79 @@
+$ErrorActionPreference = "Stop"
+
+$scriptpath = $MyInvocation.MyCommand.Path
+$dir = Split-Path $scriptpath
+
+
+try {
+    New-Variable -Name PROJECT_DIR -Value $(Resolve-Path -Path "$dir\..").Path -Option Constant
+    New-Variable -Name SCRIPTS_DIR -Value $(Join-Path -Path $PROJECT_DIR -Childpath "Scripts") -Option Constant
+    New-Variable -Name VENV_DIR -Value $(Join-Path -Path $PROJECT_DIR -Childpath ".venv") -Option Constant
+
+    $packageVersions = Get-Content -Raw -Path $PROJECT_DIR\packageVersions.json | ConvertFrom-Json
+
+    New-Variable -Name DOTNET -Value $(Join-Path -Path $VENV_DIR -Childpath "dotnet") -Option Constant
+    New-Variable -Name BINARIES -Value $(Join-Path -Path $VENV_DIR -ChildPath "bin") -Option Constant
+    New-Variable -Name PACKAGES -Value $(Join-Path -Path $VENV_DIR -ChildPath "packages") -Option Constant
+    New-Variable -Name SDK_TOOLS -Value `
+        $(Join-Path -Path $PACKAGES -ChildPath "Microsoft.PowerQuery.SdkTools.$($packageVersions.PowerQuerySDKVersion)\tools") -Option Constant
+
+    [void](New-Item -ItemType Directory -Path $VENV_DIR -Force)
+    [void](New-Item -ItemType Directory -Path $BINARIES -Force)
+    [void](New-Item -ItemType Directory -Path $PACKAGES -Force)
+    [void](New-Item -ItemType Directory -Path $DOTNET -Force)
+
+    Set-Alias -Name dotnet-install -Value "$DOTNET\dotnet-install.ps1"
+    Set-Alias -Name dotnet -Value "$DOTNET\dotnet.exe"
+    Set-Alias -Name nuget -Value "$BINARIES\nuget.exe"
+    Set-Alias -Name makepqx -Value "$SDK_TOOLS\MakePQX.exe"
+    Set-Alias -Name pqtest -Value "$SDK_TOOLS\PQTest.exe"
+
+    New-Variable -Name ALIASES -Value $(Join-Path -Path $VENV_DIR -ChildPath "aliases") -Option Constant
+
+    New-Variable -Name ALIAS_LIST -Value dotnet, nuget, makepqx, pqtest -Option Constant
+    Export-Alias -Name $ALIAS_LIST -Path $ALIASES -Force
+
+
+    $config = @{
+        venvPath            = $VENV_DIR

```

Comment:
Key names have been capitalized in zhe SafeguardSessions folder. It is not a must have to rename them as the scripts will be refactored later to the root folder.

A config.json has also been introduced, preparing for the future modifications.
https://github.com/OneIdentity/SafeguardPowerBI/blob/main/SafeguardSessions/config.json